packages/migration/fixtures/7.0.0/searchbox.tsx

```diff
@@ -1,12 +1,16 @@
 import { SearchBox } from 'office-ui-fabric-react';
 
-class SearchBoxFullSizeExample extends React.Component<any, any> {
+class SearchBoxFullSizeExample extends React.Component<{}, {}> {
   public render(): JSX.Element {
     return (
       <div className="ms-SearchBoxExample">
-        <SearchBox onChange={() => console.log('onChange called')} />
-        <SearchBox onChange={() => console.log('onChange called')}>Has children...</SearchBox>
+        <SearchBox onChange={this._onChange} />
+        <SearchBox onChange={this._onChange}>Has children...</SearchBox>
       </div>
     );
   }
+
+  private _onChange = () => {
+    console.log('onChange called');
+  };
```

Comment:
These changes were also triggered because the pre-commit hooks were complaining about `tslint` issues on the use of `any` and `lambdas`.

Reply:
Fixed in #9459.